src/indexer/index.ts

```diff
@@ -92,19 +114,7 @@ export class RunestoneIndexer {
 
       // process blocks that are reorgs
       for (const blockhash of reorgBlockhashesToIndex) {
-        const blockResult = await this._rpc.getblock({ blockhash, verbosity: 2 });
-        if (blockResult.error !== null) {
-          throw blockResult.error;
-        }
-        const block = blockResult.result;
-
-        const runeUpdater = new RuneUpdater(this._network, block, true, this._storage, this._rpc);
```

Comment:
@summraznboi gm

I noticed this `reorg` parameter isn't being used in the `RuneUpdater`. Do you have plans for it? If not, I can greatly simplify this function here.

Currently, it will process all the reorg hashes before first in the loop here:

https://github.com/me-foundation/runestone-lib/blob/ff1a10d9419aedd02701c52c9a56769bcd38ffc9/src/indexer/index.ts#L116-L118

However, we don't actually need to do that. We can just set the block height further back for the latter loop to start processing from there.

Ex, here
https://github.com/me-foundation/runestone-lib/blob/ff1a10d9419aedd02701c52c9a56769bcd38ffc9/src/indexer/index.ts#L122-L132

What do you think?

Reply:
Can you provide some unit tests to ensure no regressions are occurring with this refactor?